app/controllers/contestants_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ContestantsController < ApplicationController
+    def index
+      @contestants = Contestant.includes(:projects).all
```

Comment:
I know what this code is doing but I would love to hear an explanation as to why you felt you wanted to use `includes` since it isn't something we explicitly teach.